tools/vmd/vm_test_runner.bzl

```diff
@@ -19,7 +19,7 @@
 """iOS test runner rule."""
 
 load(
-    "@build_bazel_rules_apple//apple/testing:apple_test_rules.bzl",
```

Comment:
Might need other way to load it

Reply:
Can you conditionalize load statements? I didn't think so but maybe I'm wrong

Reply:
Yeah sure - we can unify the sims in a starlark file we load in “rules ios  dependencies” - based on the api version.

Reply:
Shims*